### DISCUSSION TITLE
Use AddBicepTemplate at subscription level

### DISCUSSION THREAD

**Discussion by sveinhelge** [microsoft/aspire#5893] [Q&A] [answered]
Is there a way to override the bicep module scope when using AddBicepTemplate?

```csharp
builder
    .AddBicepTemplate("policies", "../AppHost/Templates/Bicep/somename.bicep")
    .WithParameter("name", "some-name");
```
In the Bicep file targetScope is set to 'subscription' 

From the infra folder (after running azd infra synth). The scope is set to the resource group. I want scope to be set to subscription.

```bicep
module policies 'policies/somename.bicep' = {
  name: 'policies'
  scope: rg
  params: {
    location: location
    name: 'some-name'
  }
}
```

**Comment by davidfowl** [comment, marked as answer]
No there isn't. You can file a feature request.

**Reply by sveinhelge** [reply]
I will file a feature request.

[Feature request](https://github.com/dotnet/aspire/discussions/5900)
